tutorials/RichDisplay.md

```diff
@@ -0,0 +1,87 @@
+[`RichDisplay`](https://klasa.js.org/RichDisplay.html) allows you to create a paginated menu that users of your bot will be able to browse by using reaction-based navigation.
+
+The most simple working example can achieved by this code:
+
+```javascript
+const { Command, RichDisplay } = require('klasa');
+
+module.exports = class extends Command {
+	constructor(...args) { super(...args, { description: 'Test RichDisplay' }); }
```

Comment:
All classes in Klasa must have its contents padded.
https://eslint.org/docs/rules/padded-blocks